packages/cloudvision-connector/src/Wrpc.ts

```diff
@@ -95,6 +95,8 @@ export default class Wrpc {
 
   private waitingStreams: Map<string, StreamArgs[]>;
 
+  public streamRequests: Map<EventCallback, SubscriptionIdentifier>;
```

Comment:
I think this type is off. You are pushing to the value which would make it an array

Reply:
?
this.streamRequests.set(
          callbackWithUnbind,
          (requestId as unknown) as SubscriptionIdentifier,
        );